Argo/Types/JSON.swift

```diff
@@ -54,3 +54,5 @@ public func == (lhs: JSON, rhs: JSON) -> Bool {
   default: return false
   }
 }
+
+let x :String = "Break"
```

Comment:
Colon Violation: When specifying a type, always associate the colon with the identifier<br>Name Format Violation: Variable name should be between 3 and 40 characters in length: 'x'